numpy/__init__.pyi

```diff
@@ -3342,6 +3337,7 @@ class errstate(Generic[_CallType], ContextDecorator):
         traceback: None | TracebackType,
         /,
     ) -> None: ...
+    def __call__(self, func: Callable[..., Any]) -> Callable[..., Any]: ...
```

Comment:
@BvB93 I was not sure what to do about the typing here.  We could keep subclassing here (also in Python) if that helps.